lib/user_info_provider.rb

```diff
@@ -0,0 +1,19 @@
+require 'net/http'
+require 'json'
+
+class UserInfoProvider
+    def real_name(user_id, team_id)
+      token = $teams[team_id][:bot_access_token]
+      real_name = "no user name provided"
+
+      url = URI("https://slack.com/api/users.info?token=#{token}&user=#{user_id}&pretty=1")
+
+      begin
```

Comment:
Oops, forgot to reply here.
Failing silently is okay sometimes, yeah. But what I'm trying to say here is, that since there is no `rescue` block, it will not ignore the exception. If the URL for example is not reachable it will blow up.